packages/bot/src/commands/ranking.js

```diff
@@ -0,0 +1,102 @@
+const { SlashCommandBuilder, EmbedBuilder } = require("discord.js");
+const { viewFunction } = require("../utils/viewFunction");
+const { formatDecimals } = require("../utils/formatDecimals");
+const listServerTokens = require("../graphql/queries/listServerTokens");
+
+module.exports = {
+  data: new SlashCommandBuilder()
+    .setName("ranking")
+    .setDescription("Check the ranking of transactions by server coin")
```

Comment:
Since message will be seen inside a server I think the message could be a little different:
"Check out the donation leaderbord in the server by token"